ethcore/node_filter/src/lib.rs

```diff
@@ -33,33 +38,34 @@ extern crate kvdb_memorydb;
 extern crate ethcore_io as io;
 
 use std::sync::Weak;
-use std::collections::HashMap;
-use native_contracts::PeerSet as Contract;
-use network::{NodeId, ConnectionFilter, ConnectionDirection};
+use lru_cache::LruCache;
+use parking_lot::Mutex;
```

Comment:
Why these two are in the same section as `std`? IMHO it should be 
```
std-imports

external-imports

internal-imports
```
all sorted.